cli/tools/init/mod.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+use crate::args::InitFlags;
+use crate::compat;
+use deno_core::{anyhow::Context, error::AnyError};
+use std::io::Write;
+use std::path::Path;
+
+fn create_file(
+  dir: &Path,
+  filename: &str,
+  content: &str,
+) -> Result<(), AnyError> {
+  let mut file = std::fs::OpenOptions::new()
+    .write(true)
+    .create_new(true)
+    .open(dir.join(filename))
+    .with_context(|| format!("Failed to create {} file", filename))?;
+  file.write_all(content.as_bytes())?;
+  Ok(())
```

Comment:
```rust
std::fs::write(dir.join(filename), content.as_bytes())
```

Reply:
This will overwrite existing files and doesn't provide useful error message if something goes wrong

Reply:
We dont want to overwrite a file if it already exists